test/lib/samson/git_info_test.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+require_relative '../../test_helper'
+
+SingleCov.covered!
+
+describe Samson::GitInfo do
+  describe ".version" do
+    before { Samson::GitInfo.instance_variable_set(:@version, nil) }
+
+    it "works" do
+      Samson::GitInfo.version.to_s.must_match /^\d.\d.\d$/
```

Comment:
Maybe `/^\d(.\d){2,3}$/` as the version can be 3 or 4 parts (ie `1.8.1` or `1.8.5.6`)